src/features/commentsSlice.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { createComment, getPostComments } from '../api/comments';
+import { Comment } from '../types/Comment';
+
+type Props = {
+  comments: Comment[],
+  loaded: boolean,
+  hasError: boolean,
+};
+
+const initialState: Props = {
+  comments: [],
+  loaded: false,
+  hasError: false,
+};
+
+export const loadApiComments = createAsyncThunk(
+  'comments/load', getPostComments,
+);
+
+export const addNewComment = createAsyncThunk(
+  'comments/add', createComment,
+);
+
+const commentsSlice = createSlice({
+  name: 'comments',
+  initialState,
+  reducers: {
+    setComments: (state, action: PayloadAction<number>) => {
```

Comment:
The reducer name 'setComments' might be a bit misleading, as it filters out a specific comment by its ID. Consider changing the name to 'removeComment' for better clarity 🧐